x-pack/solutions/security/plugins/security_solution/public/detection_engine/common/helpers.tsx

```diff
@@ -305,14 +305,7 @@ export const getModifiedAboutDetailsData = (rule: RuleResponse): AboutStepRuleDe
 
 export const useQuery = () => new URLSearchParams(useLocation().search);
 
-export type PrePackagedRuleInstallationStatus =
```

Comment:
Are those deletion of the type is intentional?

Reply:
Yes, my IDE indicates when things are not used anywhere and this type was just plain unused... I can put it back but not sure why we would want to?

Reply:
I kept the removal in this PR for now, let me know if I should undo the change.